applications/motion_estimator_surf_test.cpp

```diff
@@ -0,0 +1,121 @@
+/* 
+ *  Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2016, Natalnet Laboratory for Perceptual Robotics
```

Comment:
Change copyright year; include authors

Reply:
The file _applications/motion_estimator_surf_test.cpp_ was deleted.